src/styles/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+
+  width: ($cell-size * 7) + 6px;
+
+  @for $index from 1 through 31 {
+    &__day:nth-child(#{$index}) {
+      box-sizing: border-box;
+      border: 1px solid black;
+      width: $cell-size;
+      height: $cell-size;
+
+      font-family: "Arial", sans-serif;
```

Comment:
move this to `body` selector

Reply:
I don't understand what I must to remove and why, width and height to body selector ? It stop working at all.